### PR TITLE
refactor!: remove onchain code

### LINKED ISSUE #23: Migrate node to cartesi/rollups-node
## 📚 Context

As the node unit, we want more control over the git repository.

## ✔️ Solution

Migrate the node codebase to https://github.com/cartesi/rollups-node

## 📈 Subtasks

- [x] Create the new repository
- [x] Move the code to the new repository
- [x] Create milestones in the new repository
- [x] Migrate issues to the new repository
- [x] Remove offchain code from the old repository
- [ ] Remove onchain code from the new repository

### LINKED ISSUE #66: Broken link in README
## 🙂 Expected behavior

Link in README should point to actual file.

## 🫠 Actual behavior

Link points to missing file.

## 🧪 Minimal test case

1. Look for the link in the README named "COPYING"
2. Click on it
3. It should lead to a 404 page

## 🌎 Environment

Any internet browser.

## ✔️ Possible solutions

- Delete the link
- Make the link point to elsewhere
- Create a COPYING file

### PR DESCRIPTION
**Pull request by gligneul**
Close #66
Close #23

### CODE REVIEW

**Comment by marcelstanley** [comment]
Shouldn't we also move up all directories under `offchain` and remove it?

File: AUTHORS [hunk 1]
```diff
@@ -1,5 +1,3 @@
 Gabriel de Quadros Ligneul <8294320+gligneul@users.noreply.github.com>
-Pedro Argento <pedro.argento@gmail.com>
 Guilherme Dantas <guidanoli@proton.me>
-Daniel Ortega Berja <daniel.ortega@dlinn.tech>
 Zehui Zheng <work996mail@gmail.com>
```
Comment (torives) [inline AUTHORS:3]: Pedro hasn't submitted any commits to the repo, but Daniel did. Shouldn't he remain as an author?

File: README.md [hunk 2]
```diff
@@ -1,276 +1,147 @@
...
-    click ERC20 href "https://eips.ethereum.org/EIPS/eip-20"
-    click ERC721 href "https://eips.ethereum.org/EIPS/eip-721"
-    click ERC1155 href "https://eips.ethereum.org/EIPS/eip-1155"
-```
-
-## Input Box
+# Cartesi Node Reference Implementation
 
-This module is the one responsible for receiving inputs from users that want to interact with DApps. For each DApp, the module keeps an append-only list of hashes. Each hash is derived from the input and some metadata, such as the input sender, and the block timestamp. All the data needed to recontstruct a hash is available forever on-chain. As a result, one does not need to trust data providers in order to sync the off-chain machine with the latest input. Note that this module is completely permissionless, and we leave the off-chain machine to judge whether an input is valid or not.
+The [Cartesi Node](https://docs.cartesi.io/cartesi-rollups/main-concepts/#cartesi-nodes) is the part of the [Cartesi Rollups Framework](https://docs.cartesi.io/cartesi-rollups/overview/) responsible for handling the communication between the on-chain smart contracts an the [Cartesi Machine](https://docs.cartesi.io/machine/intro/).
```
Comment (torives) [inline README.md:3]:
```suggestion
The [Cartesi Node](https://docs.cartesi.io/cartesi-rollups/main-concepts/#cartesi-nodes) is the part of the [Cartesi Rollups Framework](https://docs.cartesi.io/cartesi-rollups/overview/) responsible for handling the communication between the on-chain smart contracts and the [Cartesi Machine](https://docs.cartesi.io/machine/intro/).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 - Removed configurable inspect-server path prefix
+- Remove onchain code from node repository
```
Comment (torives) [inline CHANGELOG.md:18]:
```suggestion
- Remove on-chain code from the node repository
```

**Comment by gligneul** [comment]
> Shouldn't we also move up all directories under `offchain` and then remove it?

I'm not too fond of this idea because we plan to add Go to the project. I want to avoid mixing up the Rust code with the Go code. I could rename the offchain dir to `legacy`. @cartesi/node-unit What do you think?

File: AUTHORS (hunk 1 above)
Comment (gligneul) [reply AUTHORS:3]: I removed him because he didn't submit any changes to the remaining files of the repo.

**Comment by torives** [comment]
> > Shouldn't we also move up all directories under `offchain` and then remove it?
> 
> I'm not too fond of this idea because we plan to add Go to the project. I want to avoid mixing up the Rust code with the Go code. I could rename the offchain dir to `legacy`. @cartesi/node-unit What do you think?

Then I guess Go wouldn't be an experiment anymore: it's all-in. I have no problems with that.

**Review by marcelstanley — DISMISSED** [review]

**Comment by gligneul** [comment]
Rebased on top of https://github.com/cartesi/rollups-node/pull/84

**Review by torives — COMMENTED** [review]
Let me rename it to `legacy` on the first PR of #80, it'll make more sense 😉

**Review by torives — DISMISSED** [review]

**Review by marcelstanley — DISMISSED** [review]

**Review by torives — DISMISSED** [review]

**Review by marcelstanley — APPROVED** [review]

**Review by torives — APPROVED** [review]

**Pull request merged**